libc/src/sys/mman/shm_open.h

```diff
@@ -0,0 +1,20 @@
+//===-- Implementation header for shm_open function -------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_SYS_MMAN_SHM_OPEN_H
+#define LLVM_LIBC_SRC_SYS_MMAN_SHM_OPEN_H
+
+#include <fcntl.h> // For mode_t.
```

Comment:
please add mode_t to include/llvm-libc-macros/linux/fcntl-macros.h then use that.

Reply:
Sorry for this error inclusion. Since I have added `mode_t` to `<sys/mman.h>`, I should have used that header.